docs/rules/padding-line-between-statements.md

```diff
@@ -0,0 +1,235 @@
+# Require or disallow padding lines between statements (padding-line-between-statements)
+
+This rule requires or disallows blank lines between the given 2 kinds of statements.
+Properly blank lines help developers to understand the code.
+
+For example, the following configuration requires a blank line between a variable declaration and a `return` statement.
+
+```js
+/*eslint padding-line-between-statements: [
+    "error",
+    { blankline: "always", prev: "var", next: "return" }
+]*/
+
+function foo() {
+    var a = 1;
+
+    return a;
+}
+```
+
+## Rule Details
+
+This rule does nothing if no configuration.
+
+A configuration is an object which has 3 properties; `blankline`, `prev` and `next`. For example, `{ blankline: "always", prev: "var", next: "return" }` is meaning "it requires one or more blank lines between a variable declaration and a `return` statement."
```

Comment:
Can we rename the `blankline` property to something like `blankLine` or `blank-line`?